.github/workflows/python-django-app.yml

```diff
@@ -35,7 +35,7 @@ jobs:
           pip install -r dev-requirements.txt
       - name: Lint with pylint
         run: |
-          pylint --load-plugins pylint_django --django-settings-module=begintoreason_django.settings accounts/ core/ data_analysis/ educator/ think_aloud/ tutor/
+          pylint --load-plugins pylint_django --django-settings-module=begintoreason_django.settings accounts/ core/ data_analysis/ educator/ educator/ think_aloud/ tutor/
```

Comment:
You have `educator` twice...

Reply:
@rhit-hurtigna

Reply:
Sorry about that, I didn't realize I was able to control that! Should be fixed now.